src/loader/loader.c

```diff
@@ -679,17 +695,8 @@ static void inline do_load()
 	 * here again by the outer call after inval_relcache completes. This is
 	 * double-check locking, in effect.
 	 */
-	if (loaded)
-		return;
-
```

Comment:
This comment is made irrelevant by removing:
```
	if (loaded)
		return;
```

Reply:
Comment removed.